solr/core/src/java/org/apache/solr/packagemanager/SolrPackageRepository.java

```diff
@@ -0,0 +1,130 @@
+package org.apache.solr.packagemanager;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.solr.common.SolrException;
+import org.apache.solr.common.SolrException.ErrorCode;
+import org.apache.solr.packagemanager.SolrPackage.SolrPackageRelease;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
do no use `com.fasterxml.jackson.annotation.JsonProperty`

use the Solr annotation `org.apache.solr.common.annotation.JsonProperty` instead everywhere

Reply:
This mistake is extremely easy to make.  Perhaps your new json annotation processory thingy should detect `com.fasterxml.jackson.annotation.JsonProperty` and fail?

Reply:
Or configure forbiddenAPI

Reply:
+1 to forbidden API. I had suggested (privately) to Noble to put jackson's JsonProperty to forbidden APIs and he agreed. Also, I think JsonIgnore should be added to Solr's annotations, instead of my trying to use jackson's version.